c10/core/StorageImpl.h

```diff
@@ -5,8 +5,12 @@
 
 #include <c10/util/intrusive_ptr.h>
 
+#include <atomic>
+
 namespace c10 {
 
+static std::atomic<std::uint64_t> next_unique_storage_id{0};
```

Comment:
Maybe drop the `std::` for `uint64_t`.  I don't think that's common in our codebase.

Note to self: Profilo uses a `std::atomic<int64_t>`, so I assume this is safe even on 32-bit apps that are built with very old toolchains.